debug.xml

```diff
@@ -65,10 +65,15 @@
     <!-- DATABASE CONFIG -->
 
     <!--<entry key='database.driverFile'>hsqldb.jar</entry>-->
-    <entry key='database.driver'>org.h2.Driver</entry>
+    <!--<entry key='database.driver'>org.h2.Driver</entry>
     <entry key='database.url'>jdbc:h2:./target/database</entry>
     <entry key='database.user'>sa</entry>
-    <entry key='database.password'></entry>
+    <entry key='database.password'></entry>-->
+
+    <entry key='database.driver'>com.mysql.jdbc.Driver</entry>
```

Comment:
I prefer to use mysql instead of h2.
do I have to delete this entry from my debug.xml file before I commit and
them put it back?

On Wed, Jul 13, 2016 at 7:55 AM, Anton Tananaev <notifications@github.com>
wrote:

> In debug.xml
> <https://github.com/tananaev/traccar/pull/2104#discussion_r70564269>:
>
> >      <entry key='database.url'>jdbc:h2:./target/database</entry>
> >      <entry key='database.user'>sa</entry>
> > -    <entry key='database.password'></entry>
> > +    <entry key='database.password'></entry>-->
> > +
> > +    <entry key='database.driver'>com.mysql.jdbc.Driver</entry>
>
> Just undo it.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/tananaev/traccar/pull/2104/files/d045c1e2174146ddc8ad8b2ef4442863f3d84a11#r70564269>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AH9fF_k1kZ-DgIAHY3zPSTlWMQiJnSYrks5qVG-3gaJpZM4JKXy->
> .
>